cli/tests/unit/streams_test.ts

```diff
@@ -179,6 +215,14 @@ Deno.test(async function readableStreamCloseWithoutRead() {
   assertEquals(await cancel, "resource closed");
 });
 
+// Close the stream without reading anything
+Deno.test(async function readableStreamCloseWithoutRead2() {
+  const cancel = deferred();
+  const rid = resourceForReadableStream(longAsyncStream(cancel));
+  core.ops.op_close(rid);
```

Comment:
Any particular reason this uses `core.ops.op_close` instead of `Deno.close()`?

Reply:
This code is more used by Rust than JS, so I wanted to use the low-level ops directly to make sure there's no additional functionality in the way.